concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -1221,13 +1230,17 @@ jobs:
             - name: paas-bootstrap
             - name: concourse-manifest
             - name: concourse-secrets
+            - name: github-oauth-secrets
           params:
             CONCOURSE_HOSTNAME: ((concourse_hostname))
             SYSTEM_DNS_ZONE_NAME: ((system_dns_zone_name))
             CONCOURSE_ATC_USER: admin
             FLY_TEAM: ((aws_account))
             FLY_TARGET: ((deploy_env))
             FLY_CMD: ./paas-bootstrap/bin/fly
+            ENABLE_GITHUB: ((enable_github))
+            GITHUB_CLIENT_ID: ((github_client_id))
+            GITHUB_CLIENT_SECRET: ((github_client_secret))
```

Comment:
Why are we passing these in here if the task is also pulling them out of the file from the state bucket? This seems to duplicate the same thing.

I don't think these 2 pipeline variables are necessary with the current implementation.

Reply:
Addressed in https://github.com/alphagov/paas-bootstrap/pull/102

Reply:
Actually, after thinking and discussing further, I think it's better that these params remain, and instead we remove the `github-oauth-secrets` resource entirely. This means that the file in the state bucket is purely managed by the `manage-github-secrets.sh` script, and is much more consistent with how the datadog secrets are managed.